docs/windows/BUILD.md

```diff
@@ -0,0 +1,533 @@
+# Building PinShare for Windows
+
+Complete guide for building PinShare Windows distribution from source.
+
+## Prerequisites
+
+### Required Tools
+
+1. **Go 1.24 or later**
+   - Download: https://golang.org/dl/
+   - Verify: `go version`
+
+2. **Node.js 20 or later**
+   - Download: https://nodejs.org/
+   - Verify: `node --version` and `npm --version`
+
+3. **Git**
+   - Download: https://git-scm.com/
+   - Verify: `git --version`
+
+### Platform-Specific Requirements
+
+#### Building on Windows
+
+**Required:**
+- **TDM-GCC** or **MinGW-w64** (for CGO/SQLite)
+  - TDM-GCC: https://jmeubank.github.io/tdm-gcc/
+  - Or MinGW-w64: https://www.mingw-w64.org/
+
+- **WiX Toolset 3.x or 4.x** (for installer)
+  - Download: https://wixtoolset.org/
+  - Add to PATH: `C:\Program Files (x86)\WiX Toolset v3.x\bin`
+
+**Optional:**
+- **Visual Studio Build Tools** (alternative to MinGW)
+  - Download: https://visualstudio.microsoft.com/downloads/
+  - Install "Desktop development with C++" workload
+
+#### Cross-Compiling from Linux
+
+**Required packages:**
+```bash
+sudo apt-get update
+sudo apt-get install -y \
+  gcc-mingw-w64-x86-64 \
+  wine64 \
+  unzip \
+  curl
+```
+
+**For Debian/Ubuntu:**
+```bash
+# Add i386 architecture for Wine
+sudo dpkg --add-architecture i386
+sudo apt-get update
+sudo apt-get install wine64 wine32
+```
+
+#### Cross-Compiling from macOS
+
+**Required:**
+```bash
+# Install Homebrew if not already installed
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+
+# Install MinGW-w64
+brew install mingw-w64
+```
+
+## Building Components
+
+### Clone Repository
+
+```bash
+git clone https://github.com/Episk-pos/PinShare.git
+cd PinShare
+git checkout infra/refactor
+```
+
+### Option 1: Build Everything (Recommended)
+
+```bash
+# On Linux/macOS
+make -f Makefile.windows windows-all
```

Comment:
How is this (and the windows equivalent below) different from build-windows.bat|sh (which is what I was using during development and testing)?

Reply:
These manual component build instructions are useful for developers who need to build individual components (e.g., just the service wrapper for debugging) without running the full build script. The build-windows.bat script calls these under the hood but having them documented helps with troubleshooting and understanding the build process.